internal/supervisor/cluster.go

```diff
@@ -264,6 +267,17 @@ func (s *ClusterSupervisor) createNodegroups(cluster *model.Cluster, logger log.
 	return s.refreshClusterMetadata(cluster, logger)
 }
 
+func (s *ClusterSupervisor) deleteNodegroup(cluster *model.Cluster, logger log.FieldLogger) string {
```

Comment:
```suggestion
func (s *ClusterSupervisor) deleteNodegroups(cluster *model.Cluster, logger log.FieldLogger) string {
```

Reply:
There's a few other logs and function names around here that use mixed naming of nodegroups and nodegroup. I would just use nodegroups for everything as it looks like this would cleanup multiple nodegroups if requested.